airrohr-firmware/airrohr-firmware.ino

```diff
@@ -1035,6 +1045,9 @@ static void readConfig() {
 			strcpy(cfg::host_influx, "");
 			cfg::send2influx = false;
 		}
+		if(cfg::gps_read && cfg::mhz19_read) { // cannot be active at the same time
+			cfg::mhz19_read = false;
```

Comment:
I wonder why we give preference to gps.. or why there is gps support at all.. seems pretty unuseful for a stationary system

Reply:
> I wonder why we give preference to gps..

This was just to not change the current behaviour.

> or why there is gps support at all.. seems pretty unuseful for a stationary system

There are some units which are not stationary and use a gps.